validation.tf

```diff
@@ -3,4 +3,8 @@ resource "aws_acm_certificate_validation" "dns_validation" {
   validation_record_fqdns = local.record_fqdns
 
   count = var.enable_validation ? 1 : 0
+
+  timeouts {
+    create = "30m"
```

Comment:
Could you help me understand how this relates to the following error, as you say in the PR comment?

```
Error: error modifying ELBv2 Listener (arn:aws:elasticloadbalancing:us-east-1:949693767820:listener/app/blackforest-1-snp/300d2ea2c7058bad/74265b884aaea704): UnsupportedCertificate: The certificate 'arn:aws:acm:us-east-1:949693767820:certificate/9d1f4695-625d-49e2-8490-cd582706ff81' must have a fully-qualified domain name, a supported signature, and a supported key size.
```

Also, it appears to me that the default timeout is `75m`, which is greater than `30m`: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation#timeouts

...is it your intention to _lower_ or _raise_ the timeout?

Reply:
It was a misunderstanding on my end, I will close this PR since it doesn't appear to resolve the original issue I've been testing for.